### ISSUE TITLE
Upgrade to Sufia 7.1.0

### ISSUE THREAD

**Issue by pgwillia** [ualbertalib/Hydranorth2#17]

**Comment by pgwillia** [comment]
Solved by hydra-works 0.13.0

```
NameError in CurationConcerns::GenericWorksController#create
uninitialized constant ActiveFedora::OmDatastream
Extracted source (around line #2):

module Hydra::Works::Characterization
  class FitsDatastream < ActiveFedora::OmDatastream
    include OM::XML::Document

    set_terminology do |t|
      t.root(path: 'fits',

hydra-works (0.12.0) lib/hydra/works/characterization/fits_datastream.rb:2:in `<module:Characterization>'
hydra-works (0.12.0) lib/hydra/works/characterization/fits_datastream.rb:1:in `<top (required)>'
hydra-works (0.12.0) lib/hydra/works/services/characterization_service.rb:19:in `initialize'
hydra-works (0.12.0) lib/hydra/works/services/characterization_service.rb:10:in `new'
hydra-works (0.12.0) lib/hydra/works/services/characterization_service.rb:10:in `run'
curation_concerns (1.4.0) app/jobs/characterize_job.rb:9:in `perform'
```

**Comment by pgwillia** [comment]
Usurped by #20
